cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -21,6 +21,9 @@ import (
 
 	"math"
 
+	"bytes"
```

Comment:
Can you remove empty line above? All those imports are from standard library, so no need to split them

Reply:
done